### PR TITLE
Verify Tiger Config

### PR DESCRIPTION
**Pull request by joshwalawender**
Test to check whether the default.tigervnc file was created with the correct RemoteResize value.

### CODE REVIEW

File: test_tigervnc.py [hunk 1]
```diff
@@ -18,23 +19,24 @@
...
-            log.error('Could not find RemoteResize setting')
+            kvl.log.error('Could not find RemoteResize setting')
             assert RRsearch is not None
         else:
             remote_resize_value  = int(RRsearch.group(1))
-            log.info(f'Found RemoteResize set to {remote_resize_value}')
-            assert remote_resize_value == 1
+            kvl.log.info(f'Found RemoteResize set to {remote_resize_value}')
+            if remote_resize_value !=0:
+                kvl.log.error('RemoteResize must be set to 1')
```
Comment (joshjriley) [inline test_tigervnc.py]: I think this error msg should say 'must be set to 0'.

File: test_tigervnc.py [hunk 2]
```diff
@@ -18,23 +19,24 @@
...
+    kvl.log.info(f'Checking VNC viewer: {" ".join(cmd)}')
+    result = subprocess.run(cmd, capture_output=True)
+    output = result.stdout.decode() + '\n' + result.stderr.decode()
 
-    if re.search(r'TigerVNC', help_STDOUT.decode()):
+    if re.search(r'TigerVNC', output):
         # VNC Viewer is TigerVNC, so we need to verify the RemoteResize config
-        with open(Path('~/.vnc/default.tigervnc').expanduser()) as FO:
+        tigervnc_config_file = Path('~/.vnc/default.tigervnc').expanduser()
+        assert tigervnc_config_file.exists()
```
Comment (joshjriley) [inline test_tigervnc.py:39]: When this fails you don't get a concise message as to why in the pytest output.  I think a log.error here might be good like the others below.

**Review by joshjriley — CHANGES_REQUESTED** [review]
Just one typo in an error message.  Otherwise, works good.  Tested all conditions.

This is sort of unrelated, but I think pytest output is not very end-user friendly.  It seems like it is designed for developers.  As an end user I would prefer something more concise like one line each test:
- Checking file existence: OK
- Checking resize value: ERROR: Remote resize must be set to 0.

Just my two cents.

**Comment by joshwalawender** [comment]
I agree about pytest.  I used it because it was easy and could parameterize some of the connection tests.  It's not great for users.  We can look at swapping it out later.

I refactored this in to two tests and improved the log messages.

**Pull request merged**